CMakeLists.txt

```diff
@@ -144,6 +148,10 @@ set(CM_PROJECTS_ALL)
 # Include all optional projects. These are the ones user can pick and choose.
 # Some of these allow using system libraries too.
 include(zlib)
+include(freetype)
+include(fontconfig)
```

Comment:
Probably not.  I had to add an `if (NOT WIN32)` block around both fontconfig and freetype.  You could also add a dummy project in the main projects directory...